db/migrate/20240206115152_add_tests_title_null_constraint.rb

```diff
@@ -0,0 +1,5 @@
+class AddTestsTitleNullConstraint < ActiveRecord::Migration[7.1]
+  def change
+    change_column_null(:tests, :title, false)
```

Comment:
Remember, if you already have data in db in tests table, which has blank title, you will see an error while run this migration)

Reply:
This is exactly what happened =)